components/Layout.vue

```diff
@@ -145,26 +144,24 @@ function handleTocClick (headingId) {
   visibleHeadingId.value = headingId;
 };
 
-if(imageSrc) {
-  useHead({
-    meta: [
-      {
-        property: 'og:image',
-        content: imageSrc,
-      },
-      {
-        property: 'og:image:width',
-        content: '1200'
-      },
-      {
-        property: 'og:image:height',
-        content: '630'
-      },
-      {
-        name: 'twitter:card',
-        content: 'summary_large_image'
-      },
-    ],
-  });
-}
+useHead({
+  meta: [
+    {
+      property: 'og:image',
+      content: config.baseUrl + imageSrc,
```

Comment:
Please refer to the note on my PR comment regarding the reason for full URL for og:image